MoltenVK/MoltenVK/OS/MTLRenderPassDescriptor+MoltenVK.m

```diff
@@ -54,10 +54,10 @@ -(void) setRenderTargetArrayLengthMVK: (NSUInteger) length {
 
 -(NSUInteger) renderTargetWidthMVK {
 
-#if MVK_MACOS || MVK_TVOS
+#if MVK_TVOS
 	return 0;
 #endif
-#if MVK_IOS
+#if MVK_MACOS || MVK_IOS
```

Comment:
Not important, but there is a `MVK_MACOS_OR_IOS` declaration available for use too.